core/primitives/block_data.hpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_BLOCK_DATA_HPP
+#define KAGOME_CORE_PRIMITIVES_BLOCK_DATA_HPP
+
+#include "primitives/block.hpp"
+#include "primitives/justification.hpp"
+
+namespace kagome::primitives {
+
+  /**
+   * Data, describing the requested block
```

Comment:
Meaning of 'requested' block is not very clear.

Reply:
Updated